### ISSUE TITLE
Learning rate annealing

### ISSUE THREAD

**Issue by moiseshorta** [acids-ircam/RAVE#35]
Hi,

I've been trying to find the learning rate parameter for training RAVE, as I would like to perform some annealing once the training loss starts reaching a plateau, during phase 1.

Unfortunately, I don't see that parameter exposed in the `train_rave.py` and have been failing to find it.

Would it make sense to perform LR annealing for phase 1 of training or would it be better to perform towards end of training (phase 2)? Where can I set that parameter?

Thanks again!

**Comment by iamzoltan** [comment]
Hey, @caillonantoine answered that for me in a closed issue. You have to change it manually in `rave/model.py`. Goodluck!

**Comment by iamzoltan** [comment]
@caillonantoine does it make sense to add support for automatic learning rate annealing? Something else I would be interesting in taking a crack at. Thoughts?
